### PR TITLE
Create Monorepo Structure

### PR DESCRIPTION
**Pull request by felipecesr**
- Create a packages folder for workspaces
- Move Clappr to the packages folder
- Add Lerna and Yarn Workspaces to the project

### CODE REVIEW

**Comment by Drevel** [comment]
pkg:maven/org.apache.commons/commons-parent@61

File: package.json [hunk 1]
```diff
@@ -1,64 +1,15 @@
...
-    "lint:fix": "npm run lint -- --fix",
-    "release": "MINIMIZE=true rollup --config",
-    "start": "DEV=true rollup --config --watch",
-    "test": "echo 'No tests configured. This project is only for bundling the player components.'",
-    "watch": "rollup --config --watch"
-  },
-  "files": [
-    "/dist",
-    "/src"
+  "name": "clappr-project",
```
Comment (arielfurtado) [inline package.json:2]: Here isn't good consider maintain the name clappr?

**Review by arielfurtado — CHANGES_REQUESTED** [review]

**Comment by jhonatangcavalcanti** [comment]
What do you think about keeping the `clappr` name on `packages/clappr` instead of `packages/player`?
Why the `ignore-engines` flag was removed?
What about setting up the scripts like `start` on the root `package.json`?

**Comment by felipecesr** [comment]
@jhonatangcavalcanti why do we need to ignore engines?

**Comment by jhonatangcavalcanti** [comment]
We agree to remove the `ignore-engines` since it is not needed.
We've also documented how to use the `start` script on the mono repo structure.

File: package.json (hunk 1 above)
Comment (jhonatangcavalcanti) [reply package.json:2]: We've agreed to keep the `clappr-project` name for the monorepo

**Pull request merged**